src/bidiServer/MapperServer.ts

```diff
@@ -17,21 +17,30 @@
 
 import type Protocol from 'devtools-protocol';
 import WebSocket from 'ws';
-import debug from 'debug';
+import debug, {type Debugger} from 'debug';
 
 import {CdpConnection} from '../cdp/cdpConnection.js';
 import type {CdpClient} from '../cdp/cdpClient.js';
-import {LogType} from '../utils/log.js';
+import type {LogType, LogPrefix} from '../utils/log.js';
 import {WebSocketTransport} from '../utils/websocketTransport.js';
 
 const debugInternal = debug('bidi:mapper:internal');
 const debugInfo = debug('bidi:mapper:info');
-const debugMapperDebugOthers = debug('bidi:mapper:debug:others');
-const debugMapperDebugPrefix = 'bidi:mapper:debug:';
+const debugOthers = debug('bidi:mapper:debug:others');
+// Memorizes a debug creation
+const loggers = new Map<string, Debugger>();
+const getLogger = (type: LogPrefix) => {
```

Comment:
This should help with speed imo, as internally it does some checks and setups for each call which I am not sure if calling `debug` memorizes itself.